{
    "checksum": "18a32ce8e2f94df204e4b4e7738198767716a32f33d63a092f4f0ef29f6b19a4",
    "file": "sidlx_rmi_UnrecognizedNetworkException_Stub.c",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-c"
    },
    "number_of_duplicates": 2,
    "package": "babel",
    "path": "babel/1.4.0.dfsg-8.1/runtime/sidlx/sidlx_rmi_UnrecognizedNetworkException_Stub.c",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/babel/1.4.0.dfsg-8.1/",
        "metric": {
            "size": 64556
        },
        "pts_link": "https://tracker.debian.org/pkg/babel",
        "sloc": [
            [
                "ansic",
                628381
            ],
            [
                "java",
                94351
            ],
            [
                "sh",
                31700
            ],
            [
                "f90",
                17373
            ],
            [
                "cpp",
                12095
            ],
            [
                "fortran",
                9197
            ],
            [
                "python",
                8451
            ],
            [
                "makefile",
                6216
            ],
            [
                "perl",
                232
            ],
            [
                "xml",
                118
            ],
            [
                "awk",
                11
            ]
        ],
        "suites": [
            "wheezy"
        ]
    },
    "raw_url": "/data/main/b/babel/1.4.0.dfsg-8.1/runtime/sidlx/sidlx_rmi_UnrecognizedNetworkException_Stub.c",
    "stat": {
        "perms": "rw-r--r--",
        "size": 111669,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "1.4.0.dfsg-8.1"
}